Rakefile

```diff
@@ -2,6 +2,10 @@
 
 task default: :build
 
+task :serve do
+  sh 'bundle exec jekyll serve'
```

Comment:
Oh neat, I didn't think about adding the bundle bit in there too.

Reply:
Yeah did that so rubymine can run it for me with a press of a button.